### DISCUSSION TITLE
Share Grafana dashboard

### DISCUSSION THREAD

**Discussion by b0bd** [szpajder/dumpvdl2#40] [General]
Consider sharing the Grafana dashboard. It looks great, but I can't recreate it as the text image in the readme is too blurry.   Grafana definition would be ideal, but a readable screenshot would work for me to create my own.

thanks!

**Comment by szpajder** [comment]
I currently have three dashboards for VDL2. Here are the definitions:

[vdl2_per_channel_stats.json](https://github.com/szpajder/dumpvdl2/files/14466159/vdl2_per_channel_stats.json) - decoding statistics per channel (selectable with a variable)
[vdl2_aggregate_stats.json](https://github.com/szpajder/dumpvdl2/files/14466157/vdl2_aggregate_stats.json) - decoding statistics aggregated over all monitored VDL2 channels
[vdl2_message_stats.json](https://github.com/szpajder/dumpvdl2/files/14466158/vdl2_message_stats.json) - reassembly statistics, utilization of caches, etc.

They probably need some massaging before use, a few things need to be changed (like metric prefix, data source ID, etc), but I guess this can be done with a search+replace in a text editor.
